src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -52,6 +50,36 @@ const val BATCH_SIZE = 10000
 private val logger = LoggerFactory.getLogger(PostgresEntityDataQueryService::class.java)
 
 class PostgresEntityDataQueryService(private val hds: HikariDataSource) {
+
+    fun getEntitiesById(
```

Comment:
There is already a streamEntitySet version of this that you can do a toMap() on. Was there are a reason for duplicating the code?

Reply:
I didn't see a version with property type ids instead of FQNs

Reply:
Or does that exist somewhere?

Reply:
This has the same signature as ```fun streamableEntitySet(
            entitySetId: UUID,
            entityKeyIds: Set<UUID>,
            authorizedPropertyTypes: Map<UUID, PropertyType>,
            metadataOptions: Set<MetadataOption>,
            version: Optional<Long> = Optional.empty()
    )``` with parameters re-arranged.

Reply:
Is there a way in kotlin to return different types depending on a parameter or something like that? Because one needs to return a `PostgresIterable<SetMultimap<FullQualifiedName, Any>>` and the other needs to return a `PostgresIterable<SetMultimap<UUID, Any>>`

Reply:
Yes, but for expediency we can leave like this.